apps/site/assets/ts/components/Headsign.tsx

```diff
@@ -121,10 +123,11 @@ const HeadsignComponent = (props: Props): ReactElement<HTMLElement> => {
           headsign.times = headsign.times.reverse();
         }
       } else if (
-        (first > second && first !== "arriving") ||
-        second === "arrriving"
+        // add am pm stufff
+        (first.concat(firstMeridiem) > second.concat(secondMeridiem) && first !== "arriving") ||
```

Comment:
This works a lot of the time but I am concerned about edge cases. For example, `"9:11AM" > "10:04AM"` will evaluate `true` and get reversed here 🤔